src/Akeneo/Platform/Bundle/InstallerBundle/InstallStatusManager/InstallStatusManager.php

```diff
@@ -38,9 +41,17 @@ public function getPimInstallDateTime(): ?\DateTime
         return $installDatetime;
     }
 
-    /**
-     * @return bool Return a boolean about installation state of the PIM
-     */
+    public function getPimResetData(): ?array
```

Comment:
Instead of having ResetData that only have reset event why not returning directly the reset events
```suggestion
    public function getPimResetEvents(): array
```

Reply:
Yes why not, it was in the case if we wanted to store more data about reset information, but I'm fine with returning the array directly 👍